nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-process-group-configuration.js

```diff
@@ -272,6 +274,25 @@
                     $('#process-group-default-back-pressure-object-threshold').removeClass('unset').val(processGroup.defaultBackPressureObjectThreshold);
                     $('#process-group-default-back-pressure-data-size-threshold').removeClass('unset').val(processGroup.defaultBackPressureDataSizeThreshold);
 
+                    $('#process-group-log-to-own-file-combo').removeClass('unset').combo({
```

Comment:
We should use True / False here, rather than Yes / No, to be consistent with other areas of the application.